pkg/nimble/Makefile.include

```diff
@@ -82,6 +82,9 @@ endif
 ifneq (,$(filter nimble_netif,$(USEMODULE)))
   INCLUDES += -I$(RIOTPKG)/nimble/netif/include
 
+  # when using IP, the host should have a higher prio than the netif thread
```

Comment:
Now this comment makes little sense. The comment refers to the netif thread, while the `CFLAGS` below don't make any reference to it.

Reply:
Thats the problem about this construct: the prio is actually dpendent on both, the netif and the controller priority. But this is something our build system cannot express very well... In the end I decided to make it relative to the controller, because having this wrong can actually break things badly, while having it wrong in relation to the netif thread is leading merely to bad performance...

Reply:
Then this is something we should watch out for with the Kconfig integration. AFAIK constraints like this can be expressed nicely with this. /cc @leandrolanzieri.